service/runner.py

```diff
@@ -1,7 +1,53 @@
 from service import workflow
 from octopus.core import app, initialise
 
+from setproctitle import setproctitle
+import psutil, time, datetime
+
+
+def run_only_once():
+    startingproctitle = 'harvester: starting'
```

Comment:
Not a problem to block the PR, but generally startingproctitle, runningproctitle and maxwait would be module-level (outside this func) uppercase named variables. That should indicate to others not to change them (but still allow changing them in tests).